Deploy-CAM.ps1

```diff
@@ -2739,7 +2765,7 @@ function Set-RadiusSettings() {
             }
 
             if ((-not $radiusServerPort) -and (-not $ignorePrompts)) {
-                if((confirmDialog "Do you want to change your RADIUS Server Port?") -eq 'y') {
+                if((confirmDialog "RADIUS Server Port is currently $currentRadiusPort. Do you want to change your RADIUS Server Port?") -eq 'y') {
                     do {
                         try {
                             $radiusConfig.radiusServerPort = [int](Read-Host  "Enter your RADIUS Server's Listening port")
```

Comment:
Don't you get a weird error thrown if you just do the int cast this way - I had to do something like 'try int' elsewhere to get it to interpret it properly.

Reply:
Ahh there it is: [int]::TryParse(